web-client/src/views/Public/TrialsSessions/PublicTrialSessionsFilters.tsx

```diff
@@ -0,0 +1,207 @@
+import { FormGroup } from '@web-client/ustc-ui/FormGroup/FormGroup';
+import { NonPhone, Phone } from '@web-client/ustc-ui/Responsive/Responsive';
+import { PillButton } from '@web-client/ustc-ui/Button/PillButton';
+import { Select } from '@web-client/ustc-ui/Select/Selects';
+import { SelectSearch } from '@web-client/ustc-ui/Select/SelectSearch';
+import { TRIAL_SESSION_PROCEEDING_TYPES } from '@shared/business/entities/EntityConstants';
+import { props as cerebralProps } from 'cerebral';
+import { connect } from '@web-client/presenter/shared.cerebral';
+import { sequences, state } from '@web-client/presenter/app-public.cerebral';
+import React from 'react';
+
+type PublicTrialSessionsFiltersProps = {
+  ROOT: string;
+};
+
+const props = cerebralProps as unknown as PublicTrialSessionsFiltersProps;
+
+const PublicTrialSessionsFiltersDeps = {
+  displayProgressSpinnerSequence: sequences.displayProgressSpinnerSequence,
+  publicTrialSessionData: state[props.ROOT],
+  publicTrialSessionsHelper: state.publicTrialSessionsHelper,
+  updateFormValueSequence: sequences.updateFormValueSequence,
+};
```

Comment:
This doesn't need to be addressed in this PR, but I wonder how you would feel about passing these in as props rather than connecting to Cerebral? That way Cerebral has one "entry point"--into the parent component--rather than proliferating through the child components.